src/partials/section/schedule.html.eco

```diff
@@ -10,6 +10,13 @@
     </thead>
     <tbody>
 
+      <tr>
```

Comment:
🙈 🙈 🙈

Reply:
I believe you can add that to the `docpad.coffee` and just omit the speaker name and talk title.

See https://github.com/valetechtalks/valetechtalks.github.io/blob/4413e44ba489e8f6dc231b2ad115e31c1b641b21/src/partials/section/schedule.html.eco#L23-L29

Also, please change to coffee-break start at 8:30.